### PR TITLE
Updated Manual Scripts

### PR DESCRIPTION
**Pull request by kowsisoundhar12**
Updated Following Manual Scripts:
      Test_TC_DD_3_14.yaml
	Test_TC_DD_3_7.yaml
	Test_TC_DD_3_8.yaml
	Test_TC_DM_1_2.yaml
	Test_TC_IDM_2_2.yaml
	Test_TC_IDM_3_1.yaml
	Test_TC_IDM_3_2.yaml
	Test_TC_IDM_4_1.yaml
	Test_TC_IDM_4_2.yaml
	Test_TC_IDM_5_1.yaml
	Test_TC_IDM_5_2.yaml
	Test_TC_SC_4_5.yaml
	Test_TC_SC_4_9.yaml

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_DD_3_7.yaml [hunk 1]
```diff
@@ -14,25 +14,130 @@
 # Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
 
 name:
-    3.3.7. [TC-DD-3.7] Commissioning Flow - Concurrent - Negative scenario [DUT
+    3.3.7. [TC-DD-3.7] Commissioning Flow - Concurrent - Negative Scenario [DUT
     - Commissioner]
-
 config:
-    nodeId: 0x12344321
+    nodeId: "0x12344321"
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_DD_3_7.yaml:20]: Wait, why?

File: src/app/tests/suites/certification/Test_TC_DD_3_7.yaml [hunk 2]
```diff
@@ -14,25 +14,130 @@
...
-
 tests:
-    - label:
-          "Repeat steps 1.a-1.o from [TC-DD-3.5]. Force Fail any step between
-          1.a and 1.j."
+    - label: "Test Step"
       verification: |
+          Verification Steps
+      disabled: true
+    - label:
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_DD_3_7.yaml:28]: Why no blank line?

File: src/app/tests/suites/certification/Test_TC_DD_3_7.yaml [hunk 3]
```diff
@@ -14,25 +14,130 @@
...
+      verification: |4+
 
       disabled: true
+    - label:
+          "Commissioner and Commissionee discover each other and connect via the
+          discovery mode applicable for the DUT."
+      verification: |4+
 
+      disabled: true
     - label:
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_DD_3_7.yaml:40]: Why is this "disabled" grouped with the next test?  There is a lot of that in here.

File: src/app/tests/suites/certification/Test_TC_DD_3_8.yaml [hunk 4]
```diff
@@ -24,8 +24,162 @@ config:
 
 tests:
     - label:
-          "Repeat steps 1.a-1.o from [TC-DD-3.6]. Force exit any step in 1.a and
-          1.o."
+          "Commissioner has regulatory and fabric information available and has
+          accurate date, time and timezone"
+      verification: |
+
+      disabled: true
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_DD_3_8.yaml:31]: This blank line is extremely confusing.  Why is it there?

File: src/app/tests/suites/certification/Test_TC_DD_3_7.yaml (hunk 2 above)
Comment (kowsisoundhar12) [reply src/app/tests/suites/certification/Test_TC_DD_3_7.yaml:28]: i will update this

File: src/app/tests/suites/certification/Test_TC_DD_3_7.yaml (hunk 3 above)
Comment (kowsisoundhar12) [reply src/app/tests/suites/certification/Test_TC_DD_3_7.yaml:40]: i will  check and update this script

File: src/app/tests/suites/certification/Test_TC_DD_3_8.yaml (hunk 4 above)
Comment (kowsisoundhar12) [reply src/app/tests/suites/certification/Test_TC_DD_3_8.yaml:31]: this Manual scripts generated using XL sheet ,there is no verification steps in that reference xl sheet so thats why blankspace is there ,let me remove this script 

https://accounts.google.com/signin/v2/identifier?service=wise&passive=1209600&continue=https%3A%2F%2Fdocs.google.com%2Fspreadsheets%2Fd%2F1h4d4J90KAxFqUL_cXZHyM3pHjHCEXzFITkQhN2KMqDY%2Fedit&followup=https%3A%2F%2Fdocs.google.com%2Fspreadsheets%2Fd%2F1h4d4J90KAxFqUL_cXZHyM3pHjHCEXzFITkQhN2KMqDY%2Fedit&ltmpl=sheets&flowName=GlifWebSignIn&flowEntry=ServiceLogin#gid=446790162

**Review by bzbarsky-apple — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Pull request merged**